pynxtools_xrd/ikz.py

```diff
@@ -0,0 +1,548 @@
+# -*- coding: utf-8 -*-
```

Comment:
Should this not be covered by the readers package?

Reply:
Yes. This should be deleted.